function/examples/README.md

```diff
@@ -1,8 +1,37 @@
-# Examples 
+# function-examples
 
 ## Overview
 
-The following examples show how to use Function API to:
+This module contains examples showing how to use Function API.
 
-* [Initialize a workspace](./init/README.md)
-* [Apply a Function to a cluster](./apply/README.md)
+## Operators
+
+Operators allow to perform two operations on given resources: `apply` and `delete`. 
+Both operations may accept callbacks that will be executed before and/or after each operation.
+
+The library contains two implementations of operator interface: `GenericOperator` and `TriggersOperator`.
+
+Examples:
+
+* [apply operation](./cmd/operator/apply/main.go)
+* [delete operation](./cmd/operator/delete/main.go)
+
+## Callbacks
+
+Callbacks are optional functions provided by function API user to be executed before and/or after each operation. User may chain callbacks by providing multiple functions.
+
+Examples:
+
+* [pre-operation](./cmd/callbacks/pre/main.go)
+* [post-operation](./cmd/callbacks/pre/main.go)
+
+## Manager
+
+Manager allows to controll the hierarchy of the operators in `parent` - `children` relation. It handles the life cycle of objects created by the operators and tracks the references between the objects.
```

Comment:
```suggestion
The Manager allows you to control the hierarchy of the operators in the parent-children relation. It handles the life cycle of objects created by the operators and tracks the references between the objects.
```

Reply:
Although I am not sure if `Manager` should be capitalized.

Reply:
`Manager` is a type name.